src/NLog.Web.AspNetCore/LayoutRenderers/AspNetRequestTransferFormatLayoutRenderer.cs

```diff
@@ -0,0 +1,43 @@
+﻿#if ASP_NET_CORE3
+using System.Text;
+using Microsoft.AspNetCore.Connections.Features;
+using NLog.LayoutRenderers;
+using NLog.Web.Enums;
+using NLog.Web.Internal;
+
+namespace NLog.Web.LayoutRenderers
+{
+    /// <summary>
+    /// Gets the transfer format of the protocol.
+    /// Uses ITransferFormatFeature
+    /// </summary>
+    /// <remarks>
+    /// <code>${aspnet-request-transfer-format:Property=ActiveFormat}</code>
+    /// <code>${aspnet-request-transfer-format:Property=SupportedFormats}</code>
+    /// </remarks>
+    [LayoutRenderer("aspnet-request-transfer-format")]
+    public class AspNetRequestTransferFormatLayoutRenderer : AspNetLayoutRendererBase
```

Comment:
When is this useful ?

Reply:
Upon further reading, this one is not useful.  It only render "Binary" or "Text", but we should look at ContentType instead.  Not sure why .NET add this one.

Reply:
Removed